v2/dtos/coredata/event.go

```diff
@@ -0,0 +1,117 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package coredata
```

Comment:
My suggestion is to not include a service name in the package path. Instead, I would separate requests and responses agnostic of service. 
```
v2/dtos/requests
v2/dtos/responses
```

We have been discussing changing the name of  the CoreData service to SupportData which would then require us to change all of these package references, if we could do it at all since it would most likely be considered backwards incompatible.

Reply:
Updated as suggested, thanks.